src/Administration/Resources/app/administration/src/app/component/form/sw-custom-field-set-renderer/index.js

```diff
@@ -326,18 +329,69 @@ Component.register('sw-custom-field-set-renderer', {
             };
         },
 
-        onChangeCustomFieldSets(value, updateFn) {
-            if (this.visibleCustomFieldSets.length > 0 && this.$refs.tabComponent) {
-                // Reset state of tab component if custom field selection changes
-                this.$nextTick(() => {
-                    this.$refs.tabComponent.mountedComponent();
-                    this.$refs.tabComponent.setActiveItem(this.visibleCustomFieldSets[0]);
-                });
-            }
+        customFieldSetCriteriaById() {
+            const criteria = new Criteria(1, 1);
+            const customFieldsCriteria = new Criteria(1);
+
+            customFieldsCriteria.addSorting(Criteria.sort('config.customFieldsPosition'));
+
+            criteria
+                .addAssociation('customFields', customFieldsCriteria);
 
-            if (typeof updateFn === 'function') {
-                updateFn(value);
+            return criteria;
+        },
+
+        refreshCustomFields(sets, cb) {
```

Comment:
Could you please write tests for this? The current jest tests are failing.

Reply:
done